src/Microsoft.SqlTools.ServiceLayer/Metadata/Contracts/ObjectMetadata.cs

```diff
@@ -3,34 +3,124 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+using Microsoft.SqlServer.Management.Smo;
+
 namespace Microsoft.SqlTools.ServiceLayer.Metadata.Contracts
 {
     /// <summary>
     /// Metadata type enumeration
     /// </summary>
     public enum MetadataType
     {
-        Table = 0,
-        View = 1,
-        SProc = 2,
-        Function = 3,
-        Schema = 4,
-        Database = 5
+        Unknown = 0,
```

Comment:
I think other extensions, such as the Postgres provider, have a dependency on this.  If we're going to change we should probably update all the providers at the same time.